### PR TITLE
Update Unicode data to version 11

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by jkotas** [comment]
@dotnet-bot test this please

**Comment by GrabYourPitchforks** [comment]
I'd love for us to add an API to get the version of the Unicode spec that's burned in to the product. Otherwise LGTM.

**Comment by tarekgh** [comment]
System.Globalization.Tests.CharUnicodeInfoGetUnicodeCategoryTests.GetUnicodeCategory is expected to fail in this PR. I already have other PR in corefx for fixing this test.

**Comment by jkotas** [comment]
Disable the affected tests in https://github.com/dotnet/coreclr/blob/master/tests/CoreFX/CoreFX.issues.json . We want to keep the CI green.

**Comment by jkotas** [comment]
Should we move this file to shared CoreLib partition as part of this and update the data in CoreRT/ProjectN as well?

File: src/System.Private.CoreLib/System.Private.CoreLib.csproj [hunk 1]
```diff
@@ -372,7 +372,6 @@
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Metadata\AssemblyExtensions.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(BclSourcesRoot)\System\Globalization\CharUnicodeInfoData.cs" />
```
Comment (jkotas) [inline src/System.Private.CoreLib/System.Private.CoreLib.csproj]: Could you please delete this file as well?

Comment (tarekgh) [reply src/System.Private.CoreLib/System.Private.CoreLib.csproj]: interesting, I did move command in PS windows and I thought it deleted the original file but it didn't. anyway, I'll fix that. Thanks for the catch.

**Comment by jkotas** [comment]
> Disable the affected tests in https://github.com/dotnet/coreclr/blob/master/tests/CoreFX/CoreFX.issues.json

You will need to do this before merging this PR...

**Comment by tarekgh** [comment]
@jkotas does CI pick the tests from corefx repro master? or it pick it up from some other place? I am asking because I have fixed the test on corefx which should work fine now.

**Comment by jkotas** [comment]
The CI picks up a fixed snapshot that is several weeks old. Your fix won't propagate into this snapshot anytime soon.

**Comment by tarekgh** [comment]
Test Windows_NT arm Cross Checked Innerloop Build and Test please

**Comment by tarekgh** [comment]
test OSX10.12 x64 Checked Innerloop Build and Test
test Windows_NT arm Cross Checked Innerloop Build and Test

**Comment by tarekgh** [comment]
test Windows_NT arm Cross Checked Innerloop Build and Test

**Comment by tarekgh** [comment]
test Ubuntu arm Cross Checked Innerloop Build and Test

**Comment by tarekgh** [comment]
test Windows_NT arm Cross Checked

**Comment by tarekgh** [comment]
test Windows_NT arm Cross Checked Innerloop Build and Test

**Comment by tarekgh** [comment]
anyone knows how to restart "Windows_NT arm Cross Checked Innerloop Build and Test "?

**Comment by tarekgh** [comment]
@dotnet/dnceng do you know how I can restart "Windows_NT arm Cross Checked Innerloop Build and Test " leg?

**Comment by mmitche** [comment]
@dotnet-bot test Windows_NT arm Cross Checked Innerloop Build and Test

**Comment by tarekgh** [comment]
@mmitche thanks for help. just wondering why I can start other legs without specifying "@dotnet-bot" while I cannot do it for "Windows_NT arm Cross Checked Innerloop Build and Test". I am just curious to know :-)

**Comment by mmitche** [comment]
@dotnet-bot is just a convention, not required.  It's generally just "test <leg name>"

**Comment by tarekgh** [comment]
I am pretty sure I tried "test" a couple of times with this leg and didn't work.

**Pull request merged**